python_modules/libraries/dagster-dg/dagster_dg_tests/utils.py

```diff
@@ -20,24 +20,24 @@
 
 
 @contextmanager
-def isolated_example_deployment_foo(runner: Union[CliRunner, "ProxyRunner"]) -> Iterator[None]:
+def isolated_example_deployment_foo(runner: Union[CliRunner, "ProxyRunner"]) -> Iterator[Path]:
     runner = ProxyRunner(runner) if isinstance(runner, CliRunner) else runner
-    with runner.isolated_filesystem(), clear_module_from_cache("foo_bar"):
+    with runner.isolated_filesystem() as path, clear_module_from_cache("foo_bar"):
         runner.invoke("deployment", "scaffold", "foo")
         with pushd("foo"):
-            yield
+            yield path / "foo"
```

Comment:
nit: since `runner.isolated_filesystem` changes the cwd, why not use `Path.cwd() / "foo"`? Part of the code here already depends on changed cwd so I feel like having this separate `path` channel makes it less clear.